vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -121,9 +121,9 @@ ent prj pbs-init
 ``` sh
 ent prj pbs-publish
 ```
-3. Create a Kubernetes Custom Resource and apply it to your Entando instance. You can modify the target namespace parameter (`-n`) if you changed it from the default.
+3. Register the bundle into the ECR registry.
 ``` sh
-ent prj generate-cr | ent kubectl apply -n entando -f -
+ent prj deploy
```

Comment:
I suggest adding it in the tutorial: https://dev.entando.org/v6.3/tutorials/ecr/publish-project-bundle.html#install-the-bundle-into-an-application

Because the CLI section reference this tutorial in the end. Useful to avoid repeating ourselves everywhere.

Reply:
This is the CLI reference doc so it should also be here, maybe a new section under prj. It's a really important new capability.